.appveyor.yml

```diff
@@ -0,0 +1,48 @@
+# See https://www.appveyor.com/docs/getting-started-with-appveyor-for-linux/
+skip_branch_with_pr: true
+
+# AppVeyor will run on the output branch commits unless the first line of the
+# commit message contains '[ci skip]'
+# Enable 'Do not build on "Push" events' in the AppVeyor project settings
+# to only build commits from pull requests
+branches:
+  only:
+    - master
+
+only_commits:
+  files:
+    - .appveyor.yml
+    - content/
+
+image: ubuntu
+services:
+  - docker
+
+install:
+  - source ci/install.sh
+  - env
+    
+test_script:
+  - bash build/build.sh
+  - MANUSCRIPT_FILENAME=manuscript-$APPVEYOR_BUILD_VERSION-${APPVEYOR_REPO_COMMIT:0:7}.pdf
+  - cp output/manuscript.pdf $MANUSCRIPT_FILENAME
+  - appveyor PushArtifact $MANUSCRIPT_FILENAME
+
+build: off
+
+cache:
+  - ci/cache
+
+on_success:
+  - echo "Artifacts available from https://ci.appveyor.com/project/$APPVEYOR_REPO_NAME/builds/$APPVEYOR_BUILD_ID/artifacts"
+  - echo "Updated PDF available from https://ci.appveyor.com/api/buildjobs/$APPVEYOR_JOB_ID/artifacts/$MANUSCRIPT_FILENAME"
+
+# The following lines can be safely deleted, which will disable AppVeyorBot
+# notifications in GitHub pull requests
+# Notifications use Mustache templates http://mustache.github.io/mustache.5.html
+# See https://www.appveyor.com/docs/notifications/#customizing-message-template
+# for available variables
+notifications:
+  - provider: GitHubPullRequest
+    template: "{{#passed}}:white_check_mark:{{/passed}}{{#failed}}:x:{{/failed}} [Build {{&projectName}} {{buildVersion}} {{status}}]({{buildUrl}}) (commit {{commitUrl}} by @{{&commitAuthorUsername}})
```

Comment:
can we get this to show the triggering commit?

Reply:
The triggering commit isn't directly accessible through the webhook payload, but I found a workaround.  AppVeyor suggests using messages to transfer custom information from the build process into the payload.  We can create a message with the triggering commit and then use that message in the pull request notification.

The end result is a notification like https://github.com/agitter/my-manuscript/pull/11#issuecomment-518842928

What do you think about that?  A side effect is a message in the build that doesn't mean much on its own https://ci.appveyor.com/project/agitter/my-manuscript/builds/26515001/messages

Reply:
> What do you think about that? A side effect is a message in the build that doesn't mean much on its own

Potentially we can move more of the github comment text to the message to make it more understandable in the build context. Let me propose an ideal notification text.

Reply:
That would help.  However, I didn't find the equivalent of the webhook's `commitAuthorUsername` in the environment variables, and I like reporting that username instead of the `APPVEYOR_REPO_COMMIT_AUTHOR`.

Reply:
One downside of these PR comments is that they will end up triggering a bunch of email notifications, which I don't think are useful (since the commit email notification implies AppVeyor will start). However, doing an `@` mention probably won't increase the number of emails in most cases, because the user mentioned is likely already subscribed to the PR. So OKAY with me.